_includes/components/universal/Tooltip/scripts.js

```diff
@@ -35,6 +35,8 @@ function scripts() {
 
 
   const attachEventListeners = (tooltipNode) => {
+    console.log(tooltipNode);
```

Comment:
Necessary?

Reply:
Yeah, sorry about that, it's been a long couple of weeks @longhotsummer :(